arches/app/media/css/accessibility.css

```diff
@@ -71,4 +71,48 @@
 .link-spacing:hover {
     color: #7AD02F;
     text-decoration: underline;
+}
+
+.mainnav-sm #mainnav-menu *:focus,
+.mainnav-sm #mainnav-menu a:hover {
+    text-decoration: none;
+}
+
+#mainnav-menu a {
+    transition: none;
+}
+
+#mainnav-menu *:focus,
+#mainnav-menu a:hover,
+.base-page .navbar-brand:focus,
+.collapsed-menu-trigger:focus {
+    -webkit-box-shadow: inset 5px 0 0 0 rgba(255,255,255,1) !important;
+    -moz-box-shadow: inset 5px 0 0 0 255,255,255,1 !important;
+    box-shadow: inset 5px 0 0 0 255,255,255,1 !important;
```

Comment:
avoid !important - also, it looks like the box-shadow style isn't being applied because the color is incorrectly specified - you need the rgba specifier.  It's currently falling back (for me, since I'm testing in chrome) to webkit-box-shadow.  You can safely remove the vendor prefix specifications for box-shadow at this point - mozilla phased out -moz-box-shadow in 2012 and in webkit around 2016.